broker/tasks/waf.py

```diff
@@ -89,6 +89,7 @@ def delete_web_acl(operation_id: str, **kwargs):
     if (
         not service_instance.dedicated_waf_web_acl_name
         or not service_instance.dedicated_waf_web_acl_id
+        or not service_instance.dedicated_waf_web_acl_arn
```

Comment:
so there are instances where we will have a name and id for the acl, but no arn, and that means ignoring the delete is fine? Or an id and an acl but no name?

Reply:
All of these properties are set at the same time by the task that creates a web ACL

https://github.com/cloud-gov/external-domain-broker/blob/main/broker/tasks/waf.py#L72-L74

So in theory, if one of them missing, they all would be, but it still felt safer to test for all three. I noticed I wasn't checking all three properties, so I added this last one

But I'm open to any way of handling this better

Reply:
I think it makes the most sense to check only for the value(s) that we'll need to do the delete (although since they're all set in the same place, it probably doesn't matter _that_ much)